infra/core/host/appserviceplan.bicep

```diff
@@ -0,0 +1,20 @@
+param name string
+param location string = resourceGroup().location
+param tags object = {}
+
+param kind string = ''
+param reserved bool = true
+param sku object
+
+resource appServicePlan 'Microsoft.Web/serverfarms@2023-12-01' = {
```

Comment:
consider bumping these versions to latest supported in 2024

Reply:
fixed